tests/console/mta.pm

```diff
@@ -20,7 +20,7 @@ sub run {
 
     assert_script_run '! rpm -q exim';
     if (is_sle('>=16')) {
-        zypper_call('in postfix');
+        zypper_call('in postfix mailx');
```

Comment:
Do we know why `mailx` is only needed on SLES16 but not on Tumbleweed? :thinking:

Reply:
No, but let me keep the code and file a bug report then